### PR TITLE
Feat: migrate current diagram to view

### PR DESCRIPTION
**Pull request by vbustamante**
Try to apply with better tracing so we can see why it's failing on tools prod.

This PR adds default view to every changeset, and geometry nodes to every component connected to that view, so we can start working on view management while the system stays functional. This will migrate all graphs to v4

### CODE REVIEW

**Review by sprutton1 — DISMISSED** [review]

**Review by sprutton1 — APPROVED** [review]

**Review by sprutton1 — APPROVED** [review]

**Review by zacharyhamm — DISMISSED** [review]

File: lib/dal/src/workspace_snapshot/migrator.rs [hunk 1]
```diff
@@ -108,16 +120,20 @@ impl SnapshotGraphMigrator {
...
+                    change_set.base_change_set_id,
                 );
 
                 let new_snapshot = match self.migrate_snapshot(ctx, snapshot_address).await {
                     Ok(new_snapshot) => new_snapshot,
                     Err(err) => {
                         let err_string = err.to_string();
-                        if err_string.contains("missing from store for node") {
+                        if err_string.contains("missing from store for node")
+                            || err_string.contains("workspace snapshot graph missing at address")
```
Comment (vbustamante) [inline lib/dal/src/workspace_snapshot/migrator.rs:135]: This is the current "fix" for the error that was blocking migrations: [trace](https://ui.honeycomb.io/system-initiative/environments/tools-production/datasets/sdf/result/D93GkGdxAyY/trace/xqS2GbSVofM?fields[]=s_name&fields[]=s_serviceName&span=a25281cc6e2b076c)

Delete changesets with unreachable workspace snapshots

**Review by jhelwig — DISMISSED** [review]

**Review by zacharyhamm — DISMISSED** [review]

**Review by zacharyhamm — APPROVED** [review]
🪨 🇮🇹

**Pull request merged**